trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/deflect_control_surface.py

```diff
@@ -0,0 +1,521 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# deflect_control_surface.py
+# 
+# Created:  Jul 2022, A. Blaufox & E. Botero
+# Modified:
+#           
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+from SUAVE.Components.Wings import All_Moving_Surface
+from SUAVE.Core import Data
+
+# ----------------------------------------------------------------------
+#  Deflect Control Surface
+# ----------------------------------------------------------------------
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+def deflect_control_surface(VD,wing):
```

Comment:
how about adding an optional input that will reprocess?

Reply:
I considered that, but this function will most often be called several times since 1) each VLM_wing control surface might might be only be one part of a given control surface (if there's a wing segment that cuts the middle of a control surface, the control surface gets made into 2 VLM_wings) and 2) the user could conceivably want to deflect several control surfaces (that could each correspond to several VLM_wings) at once.